src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx

```diff
@@ -1,40 +1,65 @@
-import React, { useState } from "react";
-import { useMutation } from "@apollo/client";
+import React, { useEffect, useState } from "react";
+import { useLazyQuery, useMutation } from "@apollo/client";
 import Button, { Variant, Size } from "@leafygreen-ui/button";
 import { Popconfirm } from "antd";
 import { useAnnotationAnalytics } from "analytics";
 import { useToastContext } from "context/toast";
 import {
+  GetCreatedTicketsQuery,
+  GetCreatedTicketsQueryVariables,
   BbCreateTicketMutation,
   BbCreateTicketMutationVariables,
+  JiraTicket,
+  IssueLink,
 } from "gql/generated/types";
+
 import { FILE_JIRA_TICKET } from "gql/mutations";
+import { GET_CREATED_TICKETS } from "gql/queries";
+import { useNetworkStatus } from "hooks";
 import { ButtonWrapper } from "./BBComponents";
 
 interface FileTicketProps {
   taskId: string;
   execution: number;
-  setCreatedTicketsCount?: React.Dispatch<React.SetStateAction<number>>;
-  createdTicketsCount?: number;
+  tickets: JiraTicket[] | IssueLink[];
 }
 
 export const FileTicket: React.FC<FileTicketProps> = ({
   taskId,
   execution,
-  setCreatedTicketsCount,
-  createdTicketsCount = 0,
+  tickets,
 }) => {
   const dispatchToast = useToastContext();
+
+  const [getCreatedTicket, { startPolling, stopPolling }] = useLazyQuery<
```

Comment:
So this moves the polling logic for fetching tickets  from the parent to the child?

Reply:
yes, as soon as it files the ticket, it starts polling until it is updated.